packages/transit-vehicle-overlay/src/components/popups/VehicleTooltip/index.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Tooltip } from "react-leaflet";
+import L from "leaflet";
+
+import { transitVehicleType } from "@opentripplanner/core-utils/src/types";
+import { formatDurationWithSeconds } from "@opentripplanner/core-utils/src/time";
+import { TooltipStyle } from "../styled";
+import { linterIgnoreTheseProps } from "../../../utils";
+
+/** will show a (leaflet) map tooltip on a vehicle, showing route and update recency */
+export default function VehicleTooltip(props) {
+  const { vehicle, isTracked, direction, permanent, offset } = props;
+  linterIgnoreTheseProps(isTracked);
+
+  let rsn = vehicle.routeShortName;
+  if (rsn !== null && rsn.length <= 5) {
+    rsn = `Line ${rsn}:`;
```

Comment:
For routes that _don't_ have a short name, it appears like this 
![image](https://user-images.githubusercontent.com/2370911/81188162-ffe5eb80-8f82-11ea-8c23-8ac4c2a5fe02.png)

I.e., there is no colon at the end, but there is a route short name it adds a colon. Also, the use of the word "Line" feels inconsistent across different modes (works for rail, but to me it isn't quite a match for most bus services).

Reply:
i.e., "Line" ... that was Myleen's suggestion a year or so back.  Going to use the 'mode' (ala "BUS", "TRAM", etc...) with "Line" as a default if no route type (mode) exists.